.gitlab-ci.yml

```diff
@@ -152,9 +152,6 @@ docs:
 spellcheck:
     stage:                           workspace
     <<:                              *docker-env
-    rules:
-      - if: $CI_PIPELINE_SOURCE == "pipeline"
-        when: never
```

Comment:
What was the effect of this?

Reply:
It prevented this CI stage from running if it was run as part of a pipeline (as our CI stages are typically). I think this was introduced mistakenly, since a while ago we had issues with `cargo spellcheck`.